test/test_connection_monitoring.py

```diff
@@ -426,13 +426,15 @@ def assertRepr(self, obj):
     def test_events_repr(self):
         host = ("localhost", 27017)
         self.assertRepr(ConnectionCheckedInEvent(host, 1))
```

Comment:
Similar question from my last review. What is assertRepr? Google didn't even return results for it.

Reply:
That's so unusual. What does repr(obj) look like?

Reply:
This is pretty standard to test that repr works as expected according to the docs:
> Return a string containing a printable representation of an object. For many types, this function makes an attempt to return a string that would yield an object with the same value when passed to [eval()](https://docs.python.org/3/library/functions.html#eval);

https://docs.python.org/3/library/functions.html#repr